x-pack/legacy/plugins/siem/public/store/network/reducer.ts

```diff
@@ -151,295 +140,42 @@ export const initialNetworkState: NetworkState = {
 };
 
 export const networkReducer = reducerWithInitialState(initialNetworkState)
-  .case(setNetworkTablesActivePageToZero, state => ({
-    ...state,
-    page: {
-      ...state.page,
-      queries: setNetworkPageQueriesActivePageToZero(state),
-    },
-    details: {
-      ...state.details,
-      queries: setNetworkDetailsQueriesActivePageToZero(state),
-    },
-  }))
-  .case(updateIpDetailsTableActivePage, (state, { activePage, tableType }) => ({
-    ...state,
-    [NetworkType.details]: {
-      ...state[NetworkType.details],
-      queries: {
-        ...state[NetworkType.details].queries,
-        [tableType]: {
-          ...state[NetworkType.details].queries[tableType],
-          activePage,
-        },
-      },
-    },
-  }))
-  .case(updateNetworkPageTableActivePage, (state, { activePage, tableType }) => ({
-    ...state,
-    [NetworkType.page]: {
-      ...state[NetworkType.page],
-      queries: {
-        ...state[NetworkType.page].queries,
-        [tableType]: {
-          ...state[NetworkType.page].queries[tableType],
-          activePage,
-        },
-      },
-    },
-  }))
-  .case(updateDnsLimit, (state, { limit, networkType }) => ({
+  .case(updateNetworkTable, (state, { networkType, tableType, updates }) => ({
     ...state,
     [networkType]: {
       ...state[networkType],
       queries: {
         ...state[networkType].queries,
-        [NetworkTableType.dns]: {
-          ...state[NetworkType.page].queries.dns,
-          limit,
+        [tableType]: {
+          ...state[networkType].queries[tableType],
+          ...updates,
         },
       },
     },
   }))
-  .case(updateDnsSort, (state, { dnsSortField, networkType }) => ({
+  .case(setNetworkTablesActivePageToZero, state => ({
     ...state,
-    [networkType]: {
-      ...state[networkType],
-      queries: {
-        ...state[networkType].queries,
-        [NetworkTableType.dns]: {
-          ...state[NetworkType.page].queries.dns,
-          dnsSortField,
-        },
-      },
+    page: {
+      ...state.page,
+      queries: setNetworkPageQueriesActivePageToZero(state),
+    },
+    details: {
+      ...state.details,
+      queries: setNetworkDetailsQueriesActivePageToZero(state),
     },
   }))
-  .case(updateIsPtrIncluded, (state, { isPtrIncluded, networkType }) => ({
+  .case(setIpDetailsTablesActivePageToZero, state => ({
```

Comment:
looks like this got deleted in a merge conflict or something... we need it!

Reply:
It looks like it is not use anymore, we are using `updateNetworkTable` with this interface
```
export interface TableUpdates {
  activePage?: number;
  limit?: number;
  isPtrIncluded?: boolean;
  sort?: NetworkDnsSortField | NetworkTopTablesSortField | TlsSortField | UsersSortField;
}
```

Reply:
@XavierM thats not what i mean, im talking about the `setIpDetailsTablesActivePageToZero`, that got [deleted in your PR](https://github.com/elastic/kibana/pull/47306/files#diff-a1d3b2a1d9b4d3ebdb6a0232fa2b840bL131) but it is called from `ip_details.tsx` in the `useEffect`. At the time you did your merge it was still being called from the `top_n_flow_table`, i think thats how it got deleted